### DISCUSSION TITLE
Dropdown resetting on editing response.

### DISCUSSION THREAD

**Discussion by Kavoyaa** [Pycord-Development/pycord#669] [Q&A] [answered]
I have a help command with a dropdown menu in which you can choose a module and it edits message with an embed of all commands of that module.
It works as expected, but after editing the dropdown resets to how it initially was with nothing selected. Usually(like when just sending a new message), the dropdown will show what option we have selected, but in this case it just shows the placeholder.
Is there any way to prevent it from resetting the dropdown?

Here's how I edit the message(I just change the embed):
`await interaction.response.edit_message(embed=embed)`

This is how I want the dropdown to look like after editing the message:
![Screenshot from 2021-12-30 22-05-47](https://user-images.githubusercontent.com/78526936/147771154-b0688402-6cae-406e-ad80-f1399e0f4588.png)

This how it actually looks like after editing:
![Screenshot from 2021-12-30 22-07-15](https://user-images.githubusercontent.com/78526936/147771142-52595b7a-1e21-488f-b5a5-6e1f4f620c18.png)

**Comment by Dorukyum** [comment, marked as answer]
This is a Discord limitation, meaning it's expected behavior.

**Reply by Kavoyaa** [reply]
I've seen other bots edit messages with dropdowns but this dosen't happen with them(for example: Dank Memer).
Here's a picture(as you can see, the message is edited but the dropdown didn't reset):
![Screenshot from 2021-12-30 22-18-14](https://user-images.githubusercontent.com/78526936/147771830-d3afcda2-e21a-4722-9d97-4e33f97ea1ab.png)

**Reply by Kile** [reply]
That is because they set the option as the default when editing, which results in it being auto selected
